packages/expo-modules-core/android/src/main/cpp/JNIInjector.cpp

```diff
@@ -7,6 +7,10 @@
 #include "JavaScriptTypedArray.h"
 #include "JavaReferencesCache.h"
 
+#if IS_NEW_ARCHITECTURE_ENABLED
```

Comment:
Damn, is this really different than on iOS? 🤦‍♂️ 
On iOS it's `RCT_NEW_ARCH_ENABLED`

Reply:
it's just a convention from react-native, however we could change the defines in our cmake file.  do you want to to align ios as `RCT_NEW_ARCH_ENABLED` here?

Reply:
Yeah, I know it's from RN, I just don't like they introduced two different names for the same thing. I think it would make more sense to always use just one, the same flag on both platforms, but it's up to you.